apps/ejabberd/src/ejabberd_auth_external.erl

```diff
@@ -366,7 +366,6 @@ is_fresh_enough(TimeStampLast, CacheTime) ->
 
 %% @doc Code copied from mod_configure.erl
 %% Code copied from web/ejabberd_web_admin.erl
-%% TODO: Update time format to XEP-0202: Entity Time
```

Comment:
so, why not to update? :D

Reply:
No need to, we don't return the timestamp to the users. It's used only internally so it's more efficient to operate on the raw value.